src/components/OSCALProfile.js

```diff
@@ -58,11 +58,23 @@ export default function OSCALProfile(props) {
     };
   }, []);
 
+  let includeAll = false;
+  const includeAllControls = props.profile.imports.map(
+    (imp) => imp["include-all"]
+  );
+
+  if (includeAllControls.length > 0 && includeAllControls[0] !== undefined) {
+    includeAll = true;
+  }
   // Flatten controls and IDs into single key, value structure
   const includeControlIds = props.profile.imports
     .flatMap((imp) => imp["include-controls"])
     .flatMap((includeControl) => includeControl["with-ids"]);
 
+  const excludeControlIds = props.profile.imports
+    .flatMap((imp) => imp["exclude-controls"] ?? [])
+    .flatMap((excludeControl) => excludeControl["with-ids"] ?? []);
```

Comment:
Very clean! Can you actually apply this `?? []` "safety" to the above lines (currently 70-72) dealing with `flatMap`ing the includes?

Reply:
Thanks @kylelaker.  I will apply ?? [] to line 70 - 72.

Reply:
@mpemy have you tried this?